package.json

```diff
@@ -5,7 +5,10 @@
   "author": "Fluid Project",
   "bugs": "https://issues.fluidproject.org/browse/FLOE/component/11615",
   "homepage": "http://www.fluidproject.org/",
-  "dependencies": {},
+  "dependencies": {
+      "d3":"latest",
```

Comment:
This is a danger as per our arch list discussion at http://lists.gpii.net/pipermail/architecture/2016-March/004027.html and creates the risk the project will unexpectedly break. We should bind to specific versions of both d3 and flocking